assets.json

```diff
@@ -6084,6 +6084,46 @@
     "symbol": "ITHEUM",
     "name": "Itheum (on MultiversX)"
   },
+  {
+    "id": "elrond@AERO-458bbf",
+    "symbol": "AERO",
+    "name": "Aero (on MultiversX)"
+  },
+  {
+    "id": "elrond@BBANK-ab116b",
+    "symbol": "BBANK",
+    "name": "Bbank (on MultiversX)"
+  },
+  {
+    "id": "elrond@BMI-81000e",
+    "symbol": "BMI",
+    "name": "Bmi (on MultiversX)"
+  },
+  {
+    "id": "elrond@KRO-df97ec",
+    "symbol": "KRO",
+    "name": "Kro (on MultiversX)"
+  },
+  {
+    "id": "elrond@PROTEO-0c7311",
+    "symbol": "PROTEO",
+    "name": "Proteo (on MultiversX)"
+  },
+  {
+    "id": "elrond@SUPER-507aa6",
+    "symbol": "SUPER",
+    "name": "Super (on MultiversX)"
+  },
+  {
+    "id": "elrond@VITAL-ab7917",
+    "symbol": "VITAL",
+    "name": "Vital (on MultiversX)"
+  },
+  {
+    "id": "elrond@WAM-510e42",
+    "symbol": "WAM",
```

Comment:
There are options for developers:

1. with script
- add icons to input folder
- run the script
- if some icons are not added, probably assets.json misses info about icon
- add missing icons to assets.json and rerun the script
2. manual
- add each icon manually

I assume you have added them manually? So then there is not really a need to update the assets.json file.
We will update the readme to make this clearer.

Reply:
Updated the branch going through these steps:

- Added icons in `input/ESDT`.
- Added the missing tokens inside `assets.json`.
- Ran `main.py`.
- Reverted any modifications in `blockchain/ `except `blockchain/elrond`.